server/jobs/job_url_setter.go

```diff
@@ -0,0 +1,47 @@
+package jobs
+
+import "github.com/runatlantis/atlantis/server/events/models"
+
+//go:generate pegomock generate -m --use-experimental-model-gen --package mocks -o mocks/mock_project_job_url_generator.go ProjectJobURLGenerator
+
+// ProjectJobURLGenerator generates urls to view project's progress.
+type ProjectJobURLGenerator interface {
+	GenerateProjectJobURL(p models.ProjectCommandContext) (string, error)
+}
+
+//go:generate pegomock generate -m --use-experimental-model-gen --package mocks -o mocks/mock_project_status_updater.go ProjectStatusUpdater
+
+type ProjectStatusUpdater interface {
+	// UpdateProject sets the commit status for the project represented by
+	// ctx.
+	UpdateProject(ctx models.ProjectCommandContext, cmdName models.CommandName, status models.CommitStatus, url string) error
+}
+
+//go:generate pegomock generate -m --use-experimental-model-gen --package mocks -o mocks/mock_job_url_setter.go JobURLSetter
+
+type JobURLSetter interface {
+	// SetJobURLWithStatus sets the commit status for the project represented by
+	// ctx and updates the status with and url to a job.
+	SetJobURLWithStatus(ctx models.ProjectCommandContext, cmdName models.CommandName, status models.CommitStatus) error
+}
+
+type DefaultJobURLSetter struct {
```

Comment:
See comment about prepending default to interface names:
../atlantis/internal/team_practices/style_guide.html#interface-naming

Reply:
Moved the `JobURLSetter` interface to `project_command_runner` (where it's being used) and renamed `DefaultJobUrlSetter` to `JobURLSetter`.